manifests/m/Melodics/Melodics/2.1.6404.0/Melodics.Melodics.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Automatically updated by the winget bot at 2021/Aug/19
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Melodics.Melodics
+PackageVersion: 2.1.6404.0
+PackageLocale: en-US
+Publisher: Melodics
+PackageName: Melodics
+License: Copyright c Melodics
+ShortDescription: Melodics Setup

```

Comment:
```
ShortDescription: Melodics Setup
```
Please write a better description for this application.

Reply:
I don't think the bot is smart enough :(